src/components/Messages/index.vue

```diff
@@ -67,7 +68,7 @@ export default {
     // We can not just watch for updates of messages
     // sometime we need to react only when origin changes (whatever origin is)
     origin: {
-      type: Object,
+      type: [Object, String],
```

Comment:
I don't know if this is a good idea. If you ask me I'd prefer to see an identifier in a String format instead of a mix-and-match.